docs: changelog 2025 02 28 (#3049)
neondatabase/website: content/changelog/2025-02-28.md

---
title: Support for the AWS São Paulo region, a Neon Auth update, and more
---

## São Paulo AWS region now generally available 🇧🇷

Neon is now generally available in AWS's São Paulo region (`sa-east-1`). Create projects in the São Paulo region for lower latency access from South America and to keep your data within Brazil.

<video autoPlay playsInline muted loop width="800" height="600">
  <source type="video/mp4" src="/docs/relnotes/deploy-sao-paulo.mp4"/>
</video>

See all our supported [Regions](/docs/introduction/regions).

## Transfer projects between organizations 🔄

You can now transfer projects from one organization to another directly from the Neon Console. Organization admins can move projects to any organization they're a member of, making it easier to manage projects across different organizations.

![Transfer between organizations](/docs/relnotes/org_transfer.png)

See [Transfer projects to an organization](/docs/manage/orgs-project-transfer) to learn more.

## Create users directly from Neon Auth

> Following last week's Neon Auth Beta release, we've now added the ability to create new users directly from the Neon Console. Add test users to your project without leaving your database environment — whether you're using a Neon-managed auth project or your own auth provider.

Reviewer: "— whether you're using a Neon-managed auth project or your own auth provider." 

I would leave this out, it's a bit confusing IMO, also, the main point of this feature is to start playing with Neon Auth as early as possible, so, we might want to say something like that

Reviewer 2: changed to:
`Add test users to your project without leaving your database environment — get started trying out Neon Auth with users right away.`

![Create user in Neon Auth](/docs/guides/neon_auth_create_user.png)

See [About Neon Auth](/docs/guides/neon-auth) to learn more.

## Manage your database from Cline or Windsurf

> You can now manage your Neon database directly from Cline or Windsurf using natural language, made possible by the [Neon Model Context Protocol (MCP) Server](https://github.com/neondatabase/mcp-server-neon).

Reviewer: What about Cursor, is there a need to mention?

Reviewer 2: added a callback to last week's guides

Read more about the [Early Access Program](/docs/introduction/roadmap#join-the-neon-early-access-program).

## Postgres version updates

We updated supported Postgres versions to [14.17](https://www.postgresql.org/docs/release/14.17/), [15.12](https://www.postgresql.org/docs/release/15.12/), [16.8](https://www.postgresql.org/docs/release/16.8/), and [17.4](https://www.postgresql.org/docs/release/17.4/), respectively.

When a new minor version is available on Neon, it is applied the next time your compute restarts (for any reason). For more about how we handle Postgres version upgrades, refer to our [Postgres version support policy](/docs/postgresql/postgres-version-policy).

<details>

<summary>**Fixes & improvements**</summary>

- **Neon Console**

  - Restricted Neon Auth installation and removal to organization admins only

- **Drizzle Studio update**

  We updated the Drizzle Studio integration that powers the **Tables** page in the Neon Console to version 1.0.15. For the latest improvements and fixes, see the [Neon Drizzle Studio Integration Changelog](https://github.com/neondatabase/neon-drizzle-studio-changelog/blob/main/CHANGELOG.md).

> - **API Updates**

Reviewer 3: Hi, could you also include the changes in https://github.com/neondatabase/cloud/pull/24569? 

I believe we mentioned in previous changelogs that this change would come and I am unsure if we mentioned that buttons are now blocked less aggressively in the changelogs

---

> title: Support for the AWS São Paulo region, scheduled updates for Business Plan accounts, and more

Author: ```suggestion
title: Support for the AWS São Paulo region, a Neon Auth update, and more
```

- [Getting started with Cline and Neon Postgres MCP Server](https://neon.tech/guides/cline-mcp-neon)
- [Getting started with Windsurf and Neon Postgres MCP Server](https://neon.tech/guides/windsurf-mcp-neon)

## Scheduled updates for Business plan accounts

A few weeks ago, we announced _scheduled updates_ for Neon, which include Postgres version upgrades, security patches, and Neon feature enhancements.

Updates only take a few seconds and are applied at the scheduled time or the next time your compute restarts.

Updates for Business plan accounts will start rolling out next week. You can check for updates notices and choose a preferred update window — [learn how](/docs/manage/updates#updates-on-paid-plans).

![Paid plan updates UI](/docs/manage/paid_plan_updates.png)

_Computes larger than 8 CU or configured to scale beyond 8 CU are not updated automatically._

For more information about updates, see our [Updates documentation](/docs/manage/updates). If you have questions, please reach out to us on [Discord](https://discord.gg/92vNTzKDGp) or [contact Neon Support](https://console.neon.tech/app/projects?modal=support).

## Early Access Program now available for organizations 🔓

> Organization admins can now enable Early Access for their entire organization. Once enabled, all organization members can preview upcoming features across their organization's projects.

Author: ```suggestion
Organization admins can now enable Early Access for their entire organization. Once enabled, all organization members can preview upcoming Neon features across their organization's projects.
```

![Early Access settings for organizations](/docs/relnotes/org_early_acces.png)

> Read more about the [Early Access Program](/docs/introduction/roadmapjoin-the-neon-early-access-program).

Author: ```suggestion
Read more about [Neon's Early Access Program](/docs/introduction/roadmapjoin-the-neon-early-access-program).
```
